src/include/deepstate/DeepState.h

```diff
@@ -350,11 +358,11 @@ extern void DeepState_Begin(struct DeepState_TestInfo *info);
 /* Return the first test case to run. */
 extern struct DeepState_TestInfo *DeepState_FirstTest(void);
 
-/* Returns 1 if a failure was caught, otherwise 0. */
-extern int DeepState_CatchFail(void);
+/* Returns `true` if a failure was caught for the current test case. */
+extern bool DeepState_CatchFail(void);
```

Comment:
I have this as `int` because it's C, does changing to `bool` reduce the portability of the code?

Reply:
I don't think it commits us to any non-portability we haven't already taken on.

IIUC: `stdbool.h` is part of C99 (we are using `bool` in one other function, I think). But fixed-size types like `uint8_t` (`stdint.h`/`inttypes.h`) are _also_ properly part of C99, and we really want (and were already using) those.